capabilities/test-automation.md

```diff
@@ -1,48 +1,98 @@
+
+### Automation Isn't Meant To Replace Manual Testing
+
+Test automation is not a substitute for manual testing. Rather, automated testing _complements_ manual testing. It helps reduce the repetitive and time-consuming aspects of manually verifying system correctness. Each test carries its own cost-benefit ratio, and attempting to cover 100% of a system with automation can lead to high maintenance costs and, at times, a false sense of security. Instead, the focus should be on automating routine checks and creating tests that catch critical bugs early, reserving manual testing for more exploratory and edge-case scenarios.
+
+## Assessment
+
+To assess how mature your team or organization is in this capability, complete this short exercise.
+
+Consider the descriptions below and score yourself on the Test Automation capability. Generally, score a 1 if Test Automation is limited or completely lacking from your team or organization, a 2 if it is basic and you feel there is a LOT of room for improvement, a 3 if it is mature and you feel there is some room for improvement, and a 4 if your team or organization is exemplary in the area of Test Automation.
+
+Don't worry if the description doesn't exactly match your situation. These descriptions are meant to be examples of situations that would qualify for the associated score.
+
+* Limited: Test automation is minimal, slow, and/or unreliable. There is heavy reliance on manual testing.
+* Basic: Test automation is somewhat reliable, but is either slow or has gaps in coverage and/or value. Manual testing is required to achieve high levels of confidence.
+* Mature: Test automation is reliable, fast, and valuable, with good risk coverage. Occasional gaps are still discovered.
+* Optimized: Test automation provides a high degree of assurance that the code changes are correct, stable, and won't introduce significant issues if deployed to production. It provides a high level of assurance quickly and reliably.
+
+The number you selected represents your overall score for this capability. If you feel like your team or organization fits somewhere in between two scores, it's okay to use a decimal. For example, if you think your team or organization has somewhere between basic and mature Test Automation, then you would score a 2.5.
+
+An overall score of 2.5 or below means you'll probably gain a lot of value from experimenting with some of the supporting practices listed below. An overall score of 3 or higher means you and your team are largely proficient, or well on your way to becoming proficient, in the area of Test Automation; you would likely benefit from evaluating your scores in other capabilities.
```

Comment:
It seems odd to discuss <= 2.5 and >= 3, leaving a gap between the two numbers. Not that I'd expect an individual to score somewhere between that, but if we're averaging out an entire team or organization's responses, I could see it going somewhere in between. Can we alter how we talk about when to look deeper vs look elsewhere based on the reader's score?

Reply:
See suggested edits in 11/21 commit. We could change the 3.5 in my edit to 3. But above we describe a score of 3 as "there is still some room for improvement." 

Also, you mention something in this comment that struck me: "if we're averaging out an entire team or organization's responses..." Is the reader meant to score themselves? Or is this a team exercise and then the reader averages team scores?

On the DORA Capabilities Maturity Assessment page, it reads “choose the statement that best reflects your current experience within your team(s) or organization.” This implies the score represents **the reader’s** opinion. But, other language implies that this is a **team exercise** (“Teams can take [this assessment](https://github.com/pragmint/open-practices/blob/main/capabilities-maturity-assessment.md) “…). And your comment here implies we're averaging an entire team's responses. I think this point should be clarified. How do you usually see assessments done? As a team, or as an individual? Or both?

Reply:
I understand your confusion. This open-practices repo will likely be discovered, read, and referenced by an individual. For our clients, we run these assessments across all of their teams to get a good baseline and track our progress. So both??

As to the specific value we go with for the "needs focus threshold", I suggest we go with anything under 3. While there's room for improvement in a 3, there are likely others that scored lower and thus are better candidates for focused improvement. Does that make sense?

Reply:
Yes, it does. Edited value to 3. 

Regarding individual vs. team exercise: I suggest we keep the wording as is, for simplicity and to speak to what you feel the most likely use case is (i.e., an individual).

Reply:
Let's include the value 3. So <=3 means read further, where >3 means look elsewhere.

Reply:
Done!